backend/core/views.py

```
@@ -258,13 +287,26 @@
 # Risk Assessment
 
 
+class PerimeterFilter(df.FilterSet):
+    folder = df.ModelMultipleChoiceFilter(
+        queryset=Folder.objects.all(),
+    )
+    lc_status = df.MultipleChoiceFilter(
+        choices=Perimeter.PRJ_LC_STATUS, lookup_expr="icontains"
```

Ruff (0.8.2)

`Perimeter` may be undefined, or defined from star imports

(F405)